RecoJets/JetProducers/python/ak4PFJets_cfi.py

```diff
@@ -33,3 +33,6 @@
     doAreaFastjet = cms.bool(True),
     jetPtMin = cms.double(100.0)
     )
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+pp_on_AA_2018.toModify(ak4PFJets,src = cms.InputTag("pfNoPileUpJMEHI"), inputEtMin = cms.double(9999))
+pp_on_AA_2018.toModify(ak4PFJetsCHS,src = cms.InputTag("pfNoPileUpJMEHI"), inputEtMin = cms.double(9999))
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.